File: server/server_0.2/src/common/compress/compress_test.go
package compress

import (
	"fmt"
	"testing"
)

func Test_compress(t *testing.T) {
	s := []byte("hello")
	fmt.Println("ori", s)
	c := Compress(s, 5)
	fmt.Println(c, len(c))
	s1 := string(DeCompress(c, len(c)))
	fmt.Println(s1)

	dc := []byte{120, 218, 203, 72, 205, 201, 201, 7, 0, 6, 44, 2, 21}
	fmt.Println("hello:",DeCompress(dc, len(dc)))
}

File: server_java/src/test/cn/gocoding/common/algo/CompressTest.java
package test.cn.gocoding.common.algo; 

import cn.gocoding.common.algo.Compress;
import org.junit.Test;
import org.junit.Before; 
import org.junit.After; 

/** 
* Compress Tester. 
* 
* @author <Authors name> 
* @since <pre>三月 28, 2016</pre> 
* @version 1.0 
*/ 
public class CompressTest { 

@Before
public void before() throws Exception { 
} 

@After
public void after() throws Exception { 
} 

/** 
* 
* Method: compress() 
* 
*/ 
@Test
public void testCompress() throws Exception { 
//TODO: Test goes here...
    byte[] c = Compress.compress("hello world".getBytes());
    byte[] dc = Compress.decompress(c);
    System.out.println(dc);

    byte[] nc = new byte[] {(byte) 120, (byte)156, (byte)202, 72, (byte)205, (byte)201, (byte)201, 87, 40, (byte)207, 47, (byte)202, 73, 1, 4, 0, 0, (byte)255, (byte)255, 26, 11, 4, 93};
    byte[] ndc = Compress.decompress(nc);
    System.out.println(new String(ndc));
} 


} 
